pkg/rolie/clone.go

```diff
@@ -74,12 +89,26 @@ func (f *fetcher) cloneFeed(feed *models.Feed) error {
 }
 
 func (f *fetcher) cloneEntry(entry *models.Entry) error {
-	if len(entry.Link) > 0 {
-		if f.needsRefresh(entry) {
-			return f.storeRemoteResource(entry.Link[0].Href)
-		}
+	if len(entry.Link) < 0 {
+		return nil
 	}
-	return nil
+	if !f.filtered(entry) {
+		log.Debugf("Skipping (filtered out) download of %s", entry.Link[0].Href)
```

Comment:
@isimluk what do we mean by "Skipping (filtered out) download" ? Are we only filtering on urls? If so, I am not sure that filtering is the right sub command name.

Reply:
We skip download of rolie entries that do not match the filter that is provided.

I am definitely open to use different language if you can propose such.

Reply:
Don't have anything good especially since regex is used... so for now, `filter` it is!